src/App.tsx

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import todos from './api/todos';
+import users from './api/users';
+import { Todo } from './types/Todo';
+import { TodoList } from './component/TodoList/TodoList';
+
+const preparedTodos: Todo[] = todos.map((todo) => {
+  const todoCopy: Todo = todo;
+  const userIdFromTodos = todo.userId;
+  const userForTodo = users.find(user => user.id === userIdFromTodos);
+
+  todoCopy.user = userForTodo || null;
+
+  return todoCopy;
+});
```

Comment:
```suggestion
const preparedTodos: Todo[] = todos.map(todo => ({
  ...todo,
  user: users.find(user => user.id === todo.userId) || null,
}));
```
Набагато коротше. Але для наглядності в тебе ок